app/models/game.rb

```diff
@@ -44,4 +46,17 @@ def piece_at(x, y)
   def is_full?
     white_user_id && black_user_id
   end
+
+  def first_move
```

Comment:
Another case where my test writing is bad. :rage:  I'll make some changes tonight then, early if possible.